.github/workflows/build.yml

```diff
@@ -4,6 +4,15 @@ concurrency:
   group: test-${{ github.ref }}
   cancel-in-progress: true
 jobs:
+  lint:
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.9'
+      - run: pipx run pre-commit run --all-files
```

Comment:
Wow, TIL about `pipx`. I gather it's already installed/available to use in the target environment?

Reply:
Yes it is. I think it's part of the setup-python action.

It would be better to have a pinned version of pre-commit in a requirements file, but I didn't feel like also reworking vkgdr to use pip-compile at this point.